invenio_assets/assets/build/webpack.config.js

```diff
@@ -191,6 +191,7 @@ var webpackConfig = {
       dry: false,
       verbose: false,
       dangerouslyAllowCleanPatternsOutsideProject: true,
+      cleanStaleWebpackAssets: false,
```

Comment:
Question: did you identify what assets were stale?

Reply:
In my case they were the files from the fonts folder and img folder.

I had a quick look and the stale assets were the ones not declared in the webpack bundles, so if we want them to be picked up we would need to add them in the webpack bundles, for instance for the fonts we would need to add all the fonts in the the [invenio-theme bundle](https://github.com/inveniosoftware/invenio-theme/blob/master/invenio_theme/webpack.py#L46).

In the code of [webpack-clean-plugin](https://github.com/johnagan/clean-webpack-plugin/blob/master/src/clean-webpack-plugin.ts#L248), it's comparing the builds and removing files that were in the previous build (if the `cleanStaleWebpackAssets` option is set to True).

Reply:
It has been failing on my Linux laptop since a few months already. Other colleagues with Macs do not have this issue.

Reply:
strange, there was no new release from `ẁebpack-clean-plugin` package since two years.

Reply:
I am actually worried for implications of this. Did you check this as an alternative https://github.com/johnagan/clean-webpack-plugin/issues/116#issuecomment-474943393 ?

Reply:
I have tested both solution and unfortunately they do not solve the issue:
1) Adding `cleanAfterEveryBuildPatterns`
 ```js
cleanAfterEveryBuildPatterns: process.env.NODE_ENV === "production" ? [] : ['!*.+(svg|png|woff|woff2|eot|ttf|otf|TTF|OTF)']
```
Does not prevent the fils from being deleted.

2) Adding `watchOptions`
```js
watchOptions: {
    aggregateTimeout: 2000,
  }
```

It waits 2 seconds but the second build starts anyway, does not prevent the second build from happening